src/client/renderer/r_main.c

```diff
@@ -479,8 +478,6 @@ static void R_InitLocal(void) {
 	r_height = Cvar_Add("r_height", "0", CVAR_ARCHIVE | CVAR_R_CONTEXT, NULL);
 	r_invert = Cvar_Add("r_invert", "0", CVAR_ARCHIVE | CVAR_R_MEDIA,
 	                    "Inverts the RGB values of all world textures");
-	r_lightmap_block_size = Cvar_Add("r_lightmap_block_size", "4096", CVAR_ARCHIVE | CVAR_R_MEDIA,
-	                                 NULL);
```

Comment:
Yay, no more guessing! Do we need a max, tho? For maps like Wake (again, it's an outlier, but still..), does the engine perform better with 2x 2048 lightmaps, or 1x 4096 lightmap?